src/plugins/plugin.datavalue.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+module.exports = function(Chart) {
+
+	var helpers = Chart.helpers;
+	// var noop = helpers.noop;
+	Chart.defaults.global.valueLabel = {
+		// position: 'top',
+		fontWeight: 'normal',
+		fontColor: '#000',
+		fontSize: 10,
+		padding: {top: 0, left: 0}
+		// truncate: true
+	};
+	function DrawValue(chart, element, value, index, chartype) {
+		var opts = chart.options.valueLabel;
+		var ctx = chart.ctx;
+		var position = element.getCenterPoint();
+		// print the value on the top of the bars.
+		if (chartype === 'bar') {
```

Comment:
![image](https://cloud.githubusercontent.com/assets/11849763/25558671/0ffffe7e-2d49-11e7-8072-9ef8d3e5eacc.png)

`charttype==bar` not creating a problem in this case